lib/apiml.js

```diff
@@ -202,6 +203,93 @@ ApimlConnector.prototype = {
     });
   },
   
+  getInstanceId() {
+    return this.zluxServerEurekaClient.config.instance.instanceId;
+  },
+  
+  getZluxInstances() {
+    const eurekaClient = this.zluxServerEurekaClient;
+    return new Promise((resolve, reject) => {
+      eurekaClient.once('registryUpdated', () => {
+        const zluxInstances = eurekaClient.getInstancesByVipAddress('ZLUX');
```

Comment:
i don't know about the case sensitivity. In some parts we specify ZLUX and it appears as zlux. This may be a eureka bug.

Reply:
We register zlux-app-server with `vipAddress` `zlux` in lowercase. However, Eureka coverts it to uppercase so you can find the zlux-app-server instances only by `ZLUX` in uppercase.